### ISSUE TITLE
Error When Attempting to Start yeti_uwsgi.service 

### ISSUE THREAD

**Issue by cgalligan** [yeti-platform/yeti#82]
### Description
When running # sudo systemctl start yeti_uwsgi.service I get "Job for yeti_uwsgi.service failed because the control process exited with error code. See "systemctl status yeti_uwsgi.service" and "journalctl -xe" for details." Anyone else able to get this working?

**Comment by gaelmuller** [comment]
Could you provide the output of `systemctl status yeti_uwsgi.service` or `journalctl -xe` ?

**Comment by peasead** [comment]
Not my original issue, but I'm also having an issue with it.

### Environment

| Question         | Answer
|------------------|--------------------
| Git commit       | 029cd8e837f8b5d7145c2ab235a7e8d4cb01eb02
| OS version       | CentOS Linux release 7.3.1611 (Core)
| Browser          | Version 59.0.3071.115 (Official Build) (64-bit)

### Steps to Reproduce
1. `git clone https://github.com/peasead/yeti.git`
1. `cd yeti/extras`
1.  `sudo sh centos_bootstrap.sh`

#### Expected behavior
`uwsgi` starting.

#### Actual behavior
`uwsgi` not starting.
```
systemctl status yeti_uwsgi.service
● yeti_uwsgi.service - Yeti UWSGI server
   Loaded: loaded (/usr/lib/systemd/system/yeti_uwsgi.service; enabled; vendor preset: disabled)
   Active: failed (Result: start-limit) since Thu 2017-08-10 23:43:35 UTC; 14s ago
  Process: 13771 ExecStart=/sbin/uwsgi --socket 127.0.0.1:8000 -w yeti --callable webapp --processes 4 --threads 2 --stats 127.0.0.1:9191 (code=exited, status=1/FAILURE)
 Main PID: 13771 (code=exited, status=1/FAILURE)
   Status: "initializing uWSGI"

Aug 10 23:43:35 centos-unconfigured systemd[1]: Failed to start Yeti UWSGI server.
Aug 10 23:43:35 centos-unconfigured systemd[1]: Unit yeti_uwsgi.service entered failed state.
Aug 10 23:43:35 centos-unconfigured systemd[1]: yeti_uwsgi.service failed.
Aug 10 23:43:35 centos-unconfigured systemd[1]: yeti_uwsgi.service holdoff time over, scheduling restart.
Aug 10 23:43:35 centos-unconfigured systemd[1]: start request repeated too quickly for yeti_uwsgi.service
Aug 10 23:43:35 centos-unconfigured systemd[1]: Failed to start Yeti UWSGI server.
Aug 10 23:43:35 centos-unconfigured systemd[1]: Unit yeti_uwsgi.service entered failed state.
Aug 10 23:43:35 centos-unconfigured systemd[1]: yeti_uwsgi.service failed.
```

**Comment by peasead** [comment]
@cgalligan try this if you're running CentOS.

`yum install uwsgi-plugin-python uwsgi -y`

Your `yeti_uwsgi.service` should be changed from
```
[Unit]
Description=Yeti UWSGI server
After=mongodb.service redis.service

[Service]
Type=simple
User=yeti
WorkingDirectory=/opt/yeti
ExecStart=/sbin/uwsgi --socket 127.0.0.1:8000 -w yeti --callable webapp --processes 4 --threads 2 --stats 127.0.0.1:9191
Restart=always
KillSignal=SIGQUIT
Type=notify
StandardError=syslog
NotifyAccess=all

[Install]
WantedBy=multi-user.target
```
to
```
[Unit]
Description=Yeti UWSGI server
After=mongodb.service redis.service

[Service]
Type=simple
User=yeti
WorkingDirectory=/opt/yeti
ExecStart=/sbin/uwsgi --plugin python --socket 127.0.0.1:8000 -w yeti --callable webapp --processes 4 --threads 2 --stats 127.0.0.1:9191
Restart=always
KillSignal=SIGQUIT
Type=notify
StandardError=syslog
NotifyAccess=all

[Install]
WantedBy=multi-user.target
```

**Comment by tomchop** [comment]
Thanks for finding the solution @peasead! I assume this will be part of the PR you talked about in #125?

**Comment by peasead** [comment]
Yes sir. It's done now, but I wanted to do a few more function checks to make sure it's a whole package.

**Comment by shubhamblackstratus** [comment]
Awesome.... Thank you so much @peasead . I had never worked before on uwsgi and it's saved much time of mine.

**Comment by jalbrizio** [comment]
This should be fixed with the latest commit.

**Comment by tomchop** [comment]
Fixed in #305
